lib/src/bootloader.rs

```diff
@@ -26,3 +26,113 @@ pub(crate) fn install_via_bootupd(
         .verbose()
         .run()
 }
+
+#[context("Installing bootloader using zipl")]
+pub(crate) fn install_via_zipl(
+    device: &Utf8Path,
+    boot_uuid: &str,
+) -> Result<()> {
+
+    // Run findmnt to find the boot partition
+
+    #[derive(serde::Deserialize)]
+    struct Filesystem {
+        source: String,
+        target: String,
+    }
+
+    #[derive(serde::Deserialize)]
+    struct Findmnt {
+        filesystems: Vec<Filesystem>,
+    }
+
+    let findmnt_desc = format!("Running findmnt to identify the boot partition with UUID={boot_uuid}");
+    let findmnt_task = Task::new(findmnt_desc, "findmnt");
```

Comment:
Can you extend the existing `run_findmnt()` API we have internally for this?

Reply:
Sure. I'll create a new function in mount.rs.